app/controllers/api/v1/recordings_controller.rb

```diff
@@ -14,6 +14,12 @@ def index
         render_json(data: recordings, status: :ok, include: :formats)
       end
 
+      def destroy
+        Recording.destroy(params[:id])
+
```

Comment:
You need to actually delete the recording here

Reply:
Wouldn't we need it to be destroy in order to remove the formats that belong it as well? Or is there another reason?

Reply:
As in, you need to make a request to BigBlueButton to destroy the recording

Reply:
Will do this in next PR